pallets/liquid-staking/src/lib.rs

```diff
@@ -1040,6 +1051,14 @@ pub mod pallet {
             let account_id = Self::account_id();
             let asset: MultiAsset = (MultiLocation::here(), fees).into();
 
+            log::trace!(
+                target: "liquidstaking::ump_transact",
+                "call: {:?}, asset: {:?}, xcm_weight: {:?}",
```

Comment:
this asset information will not be very useful because it's only relaychain's native currency

Reply:
I think it conclude a fee number, which may be good for debug, as you know, this is trace level log.